include/envoy/grpc/async_client.h

```diff
@@ -119,8 +120,9 @@ template <class ResponseType> class AsyncStreamCallbacks {
    * invoked. A non-Ok status implies that stream is also locally closed and that no
    * further stream operations are permitted.
    * @param status the gRPC status.
+   * @param message the gRPC message, if present.
```

Comment:
If not present looks like this will be a blank string. Can you make this explicit in the comment?